packages/eas-cli/src/credentials/ios/utils/p12Certificate.ts

```diff
@@ -39,7 +39,23 @@ function getRawCertData(p12Buffer: Buffer | string, passwordRaw: string | null):
   const password = String(passwordRaw || '');
   const p12Der = forge.util.decode64(p12Buffer);
   const p12Asn1 = forge.asn1.fromDer(p12Der);
-  const p12 = forge.pkcs12.pkcs12FromAsn1(p12Asn1, password);
+
+  let p12: forge.pkcs12.Pkcs12Pfx;
+  try {
+    if (password) {
+      p12 = forge.pkcs12.pkcs12FromAsn1(p12Asn1, password);
+    } else {
+      p12 = forge.pkcs12.pkcs12FromAsn1(p12Asn1, false /* strict */);
```

Comment:
I'd generate a p12 with empty password and add it as a test case just as a sanity check to make sure this works. Different libraries don't behave the same with empty string passwords, unfortunately. For example, when openssl sees 'empty string' password it will try `''` to decrypt first, and if that doesn't work, it will try to not decrypt at all (null case)

Reply:
I added test and notice some additional behaviour when doing that(I'm  testing node forge at this point :D)

- `false /* strict */` is not necessary
- kesytore created by openssl should be read with  `forge.pkcs12.pkcs12FromAsn1(p12Asn1, '')`
- kesytore created by keychain should be read with  `forge.pkcs12.pkcs12FromAsn1(p12Asn1)`

keystore created with openssl will fail during build so we don't need to support that case

Reply:
btw I copied that empty openssl keystore from yuor PR